src/actors/containerization/services/wordpress/wordpress_generator/wordpress_generator.py

```diff
@@ -6,11 +6,13 @@
 
 def _execute(cmd, **kwargs):
     try:
-        return check_output(cmd, shell=True, **kwargs)
+        check_output(cmd, shell=True, **kwargs)
     except CalledProcessError as e:
         sys.stderr.write(str(e)+'\n')
         return None
 
+    return True
+
```

Comment:
returns `True` or `None`? Much better would be `True` or `False` to be consistent. What do you think? You should return `None` as negative result in case you expect returned an object.

Reply:
yeah, you're right
however, looking at this again it makes no sense to use check_output and return None at failure since we don't care about output... I'll change it to use check_call and return returncode